Documentation/contributing/development/contributing_guide.rst

```diff
@@ -198,10 +277,20 @@ Pull request review process
    required to obtain the privileges to modify GitHub labels on the pull
    request.
 
+Dedicated expectation time for each member of Janitors team: Follow the next
+steps 1 to 2 times per day. Works best if done first thing in the working day.
+
+#. Review all PRs requesting for review in `for you <https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+review-requested%3A%40me+sort%3Aupdated-asc>`_;
+
+#. If this PR was opened by a non-committer (e.g. external contributor) please
+   assign yourself to that PR and make sure to keep track the PR gets reviewed
+   and merged.
```

Comment:
```suggestion
#. If this PR was opened by a non-committer (e.g. external contributor) please
   assign yourself to that PR and make sure to keep track the PR gets reviewed
   and merged. This may extend beyond your assigned week for Janitor duty.
```

Reply:
Not sure whether to mention here as well, but my thinking is that this is best-effort and should be done by matching external contributor effort. If the contributor is responsive, we should be responsive. If the contributor is slow to respond, then we do not need to spend a lot of effort chasing them down to get their PR in.